src/main/java/luke/exceptions/IncorrectInput.java

```diff
@@ -0,0 +1,7 @@
+package luke.exceptions;
+
+public class IncorrectInput extends LukeException {
```

Comment:
You may want to name your classes with a postfix "Exception" that are extending from LukeException?